### PR TITLE
added new cross platform app and able to run single tests across plat…

### PR DESCRIPTION
**Pull request by saikrishna321**
vodQA - Sample cross platform app built using reactNative. Reason for building the cross platform app using reactNative is to get over the obstacles of automating apps built using react native and to write a single script to automate cross platform apps using Appium. vodQA app code is open sourced here and same was used for various appium workshops conducted by ThoughtWorks.

Same is available for all community activities without any license required/ involved.

**Application Repo**
https://github.com/saikrishna321/VodQAReactNative

### CODE REVIEW

File: sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/BaseCrossDriver.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import org.junit.Before;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.io.File;
+import java.io.IOException;
+
+
+public class BaseCrossDriver {
```
Comment (SrinivasanTarget) [inline sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/BaseCrossDriver.java]: @saikrishna321 Can this be renamed as BaseCrossPlatformDriver?

File: sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/AndroidDragAndDrop.java [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
-        TouchAction touchAction = new TouchAction(driver);
-        WebElement destination = driver.findElementById("io.appium.android.apis:id/drag_dot_2");
-        touchAction.longPress(calc).waitAction(6000).moveTo(destination).perform().release();
-        Thread.sleep(5000);
-        assertEquals(driver.findElementById("io.appium.android.apis:id/drag_result_text").getText(), "Dropped!");
+        login();
+        driver.findElementByAccessibilityId("dragAndDrop").click();
+        MobileElement dragMe = (MobileElement) new WebDriverWait(driver, 30)
+                .until(ExpectedConditions
+                        .elementToBeClickable(MobileBy.AccessibilityId("dragMe")));
```
Comment (SrinivasanTarget) [inline sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/AndroidDragAndDrop.java]: Indentation

File: sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/BaseCrossDriver.java (hunk 1 above)
Comment (saikrishna321) [inline sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/BaseCrossDriver.java]: Done

File: sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/AndroidDragAndDrop.java (hunk 2 above)
Comment (saikrishna321) [inline sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/AndroidDragAndDrop.java]: As per java-client style.xml

**Comment by saikrishna321** [comment]
@imurchie i have added the details to the PR.

**Comment by jlipps** [comment]
@saikrishna321 i don't see the details about the app...?

**Comment by imurchie** [comment]
@saikrishna321 The information should be _part_ of the PR, in terms of something that will be in the sample-code repo. A readme or somesuch.

**Comment by saikrishna321** [comment]
@imurchie i have added the information to the NOTICE

**Pull request merged**